packages/providers/src.ts/fallback-provider.ts

```diff
@@ -365,17 +362,12 @@ async function getRunner(config: RunningConfig, currentBlockNumber: number, meth
             if (params.blockTag && isHexString(params.blockTag)) {
                 provider = await waitForSync(config, currentBlockNumber)
             }
-            return provider[method](params.address, params.blockTag || "latest");
+            return logger.throwError("NOT_SUPPORTED", Logger.errors.UNSUPPORTED_OPERATION);
```

Comment:
I think that we should leave the original implementation for `getBalance` and `getCode` since we support those methods, however remove `getTransactionCount` from the list here